packages/kolibri-tools/lib/bundleStats.js

```diff
@@ -8,7 +8,7 @@ const buildLogging = logger.getLogger('Kolibri Build Stats');
 
 const basePort = 8889;
 
-function buildWebpack(data, index, startCallback, doneCallback) {
+function buildWebpack(data, index, startCallback, doneCallback, options) {
```

Comment:
What's the purpose of this options args? It doesn't seem to be used. The linter (if it were working properly) would have issues with these unused params!